docs/monitoring.md

```diff
@@ -1060,10 +1065,10 @@ when running in local mode.
   - hiveClientCalls.count
   - sourceCodeSize (histogram)
 
-- namespace=<Executor Plugin Class Name>
-  - Optional namespace(s). Metrics in this namespace are defined by user-supplied code, and 
-  configured using the Spark executor plugin infrastructure.
-  See also the configuration parameter `spark.executor.plugins`
+- namespace=plugin.<Plugin Class Name>
```

Comment:
I believe escape characters are needed for this, something like: namespace=plugin.\\\<Plugin Class Name>